### DISCUSSION TITLE
关于docker-compose配置中，想要增加用户名和密码，应该怎么添加变量，试了试docker配置里面的参数，添加后无效

### DISCUSSION THREAD

**Discussion by fsyllkn** [messense/aliyundrive-webdav#863] [Q&A]
关于docker-compose配置中，想要增加用户名和密码，应该怎么添加变量，试了试docker配置里面的参数，添加后无效，
把  - 'WEBDAV_AUTH_USER=xxx'
      - 'WEBDAV_AUTH_PASSWORD=xxx'
     换成
WEBDAV_USERNAME=
WEBDAV_PASSWORD=
也无效。添加的配置如下：
version: '3.3'
services:
  aliyundrive-webdav:
    container_name: aliyundrive-webdav
    restart: unless-stopped
    ports:
      - '28080:8080'
    environment:
      - 'WEBDAV_AUTH_USER=xxx'
      - 'WEBDAV_AUTH_PASSWORD=xxx'
      - 'REFRESH_TOKEN=tokentoken'
    image: messense/aliyundrive-webdav

**Comment by newcardriver** [comment]
compose这样用，亲测可以。
![dav](https://github.com/messense/aliyundrive-webdav/assets/44741731/fad1cddd-0805-4b38-8668-44784b26ef8e)
